Meta/ladybird.py

```diff
@@ -271,6 +270,8 @@ def configure_main(platform: Platform, preset: str, cc: str, cxx: str) -> Path:
 
 
 def configure_skia_jemalloc() -> list[str]:
+    import resource
```

Comment:
Please add a comment to note for ourselves why this import is here, rather than at the top